### DISCUSSION TITLE
修复 Databend aarch64 架构的支持

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#356] [Announcements]
# 修复 Databend aarch64 架构的支持

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2021/10-rust-cross-aarch64/

**Comment by hhstore** [comment]
> 你这个问题太经典了. 

- 我最近尝试在 Mac M1 下使用 Rust + FFI 编译. 遇到类似的 arm64 报错. 
- https://github.com/hhstore/blog/issues/355
- 看到你的文章, 感觉可能找到类似的解决方法. 

> 这个尴尬😓的现象, 挺普遍. 

- Rust 的工程师, 可能和 Ubuntu 的维护者, 没有交集吧. 
- 跨领域的问题, 一个领域的小 case, 对另一个领域的群众, 可能要挠头. 
- 信息鸿沟/隔行如隔山. 
- 如果不是刚好你这种交叉领域知识的人, 遇到这个问题. 可能 cross 这批开发者, 要继续挠头很久. 

> 编程细分领域过多, 过于碎片化. 

- 没有人可以掌握所有的领域知识. 
- 只能不断积累, 拓宽知识边界.
